tensorboard/data/provider.py

```diff
@@ -187,7 +203,8 @@ def read_scalars(
           plugin_name: String name of the TensorBoard plugin that created
             the data to be queried. Required.
           downsample: Integer number of steps to which to downsample the
-            results (e.g., `1000`). Required.
+            results (e.g., `1000`). See `DataProvider` class docstring
+            for details about this parameter. Required.
```

Comment:
IIUC this is not required.  It defaults to None, meaning "don't downsample" (and similarly below).

Reply:
Ah, gotcha.  Thanks for the explanation.  I was concerned that callers might not know this, so there could be `read_scalars(...)` calls that just don't mention `downsample`, but fortunately there are not.  +1 the PEP 3102 fix.